src/basic/mountpoint-util.c

```diff
@@ -664,3 +664,54 @@ unsigned long ms_nosymfollow_supported(void) {
         cache = false;
         return 0;
 }
+
+int mount_option_supported(const char *fstype, const char *key, const char *value) {
```

Comment:
Is this possible to be used in `ms_nosymfollow_supported()` ?

Reply:
no. MS_NOSYMFOLLOW is an option for the bind mount, but the mount option strings (which mount_option_supported() checks for) are per-superblock.

Hence this MS_NOSYMFOLLOW check here is for a generic kernel feature, but mount_option_supported checks a specific file system, hence the codepaths are different: one uses just fsconfig() to validate, the other uses fsmount()